### ISSUE TITLE
DailyDialog examples containing __SILENCE__ tokens

### ISSUE THREAD

**Issue by ricsinaruto** [facebookresearch/ParlAI#1659]
There's a lot of examples in the dataset where the first utterance is just __SILENCE__. Is this intended, and if so what's the purpose of it?

I feel like this __SILENCE__ token is making the learning task harder, because it's observed with a lot of responses in the data (see image below). Is there any way to use the dataset where examples containing this token are left out?

Command run:
```
python examples/train_model.py -t dailydialog -m transformer/generator --shuffle true -tr 32 --dict-maxtokens 16384 -histsz 1 -bs 256
```

![image](https://user-images.githubusercontent.com/18282017/56474810-7484df80-647f-11e9-9fd4-1b6cbed74b40.png)

**Comment by stephenroller** [comment]
This is done to be consistent with Convai2, where you need to simulate being the first person of the conversation. However, this is done entirely in the agent:

https://github.com/facebookresearch/ParlAI/blob/733b627ae456d6b11a2fc4624088a781bc6c1d03/parlai/tasks/dailydialog/agents.py#L66-L85

If you wanted to add another teacher that skipped the beginning entries, I would accept the PR.

**Comment by ricsinaruto** [comment]
Okay, thanks! I will look into it.

**Comment by ricsinaruto** [comment]
Solved by #1661
